src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write your code here
+const logo = document.getElementsByClassName('logo')[0];
+const message = document.createElement('div');
+
+message.setAttribute('class', 'message');
+
+function createPromise() {
+  return new Promise((resolve, reject) => {
+    const timeout = window.setTimeout(() => {
+      message.classList.add('error-message');
+      reject(new Error('Promise was rejected!'));
+      document.body.append(message);
```

Comment:
Appending the message element to the document body here might not be ideal since it will be appended again in the resolve handler. Consider appending it only once after setting the text content.